run/testing/grpc_ping.e2e_test.go

```diff
@@ -73,35 +73,42 @@ func TestGRPCPingService(t *testing.T) {
 	}
 
 	message := "Hello Tester"
-	resp, err := grpcRequest(pingURL.Host+":443", pingURL.String(), func(ctx context.Context, conn *grpc.ClientConn) (interface{}, error) {
-		c := pb.NewPingServiceClient(conn)
-		return c.Send(ctx, &pb.Request{
-			Message: message,
+
+	testutil.Retry(t, 10, 20*time.Second, func(r *testutil.R) {
+		resp, err := grpcRequest(pingURL.Host+":443", pingURL.String(), func(ctx context.Context, conn *grpc.ClientConn) (interface{}, error) {
+			c := pb.NewPingServiceClient(conn)
```

Comment:
question: does this client require to be `Closed()`?

Reply:
The gRPC service clients do not define a `Close` method.